laceworksdk/api/__init__.py

```diff
@@ -149,3 +151,24 @@ def __init__(self,
         self.tokens = TokenAPI(self._session)
         self.user_profile = UserProfileAPI(self._session)
         self.vulnerabilities = VulnerabilityAPI(self._session)
+
+    def set_org_level_access(self, org_level_access):
+        """
+        A method to set whether the client should use organization-level API calls.
+        """
+
+        if org_level_access is True:
+            self._session._org_level_access = True
+        else:
+            self._session._org_level_access = False
+
+        return self._session._org_level_access
```

Comment:
why return the value here? the function is called `set_` and the doc string does not indicate that a value gets returned back